apps/tenant-management-webapp/src/app/pages/admin/services/comment/comments/deleteConfirmationsView.tsx

```diff
@@ -34,11 +34,11 @@ export const DeleteConfirmationsView = ({
         title="Delete comment"
         isOpen={showDeleteConfirmation}
         content={
-          <div>
+          <DeleteConfirmation>
             <div>
-              Are you sure you wish to delete <b>{`${selectedComment.title}?`}</b>
+              Are you sure you wish to delete <p>{`${selectedComment.title}?`}</p>
```

Comment:
Hi, is this Zion's new design? I remember we need to highlight the name of the object to be deleted.